app/main.cpp

```diff
@@ -476,6 +482,8 @@ int main( int argc, char *argv[] )
   engine.rootContext()->setContextProperty( "__browseDataLayersModel", &browseLpm );
   engine.rootContext()->setContextProperty( "__activeLayer", &al );
   engine.rootContext()->setContextProperty( "__purchasing", purchasing.get() );
+  engine.rootContext()->setContextProperty( "__fieldsModel", &fm );
```

Comment:
Do we need fields model as a global variable? Can't project wizard create the fields model inside of its QML?